wallet/wallet.go

```diff
@@ -1459,7 +1460,7 @@ func (w *Wallet) SignMessage(msg string, addr dcrutil.Address) (sig []byte, err
 
 // VerifyMessage verifies that sig is a valid signature of msg and was created
 // using the secp256k1 private key for addr.
-func VerifyMessage(msg string, addr dcrutil.Address, sig []byte) (bool, error) {
+func VerifyMessage(msg string, addr dcrutil.Address, sig []byte, params *chaincfg.Params) (bool, error) {
```

Comment:
Since this one is publicly exported, it should probably be `dcrutil.AddressParams`.